backend/tests/teardown.js

```diff
@@ -0,0 +1,3 @@
+export default () => {
+  process.exit(0);
+};
```

Comment:
This was added to avoid the server being left without closing 
<img width="794" alt="image" src="https://github.com/krishnaacharyaa/wanderlust/assets/116620586/899aeed8-0182-4346-98b5-118c10f86fa4">

https://fullstackopen.com/en/part4/testing_the_backend#supertest, thanks to @shmbajaj for the link

p.s This was not related to the mongodb, monogodb's connection is handled using the below snippet
```
afterAll(async () => {
  await mongoose.disconnect();
});
```

Reply:
Hey @krishnaacharyaa awesome bro you have handled it beautifully.

Reply:
PS: Describe such info over the PR description only.

Reply:
@shmbajaj Sure